docs/components/HookTypeScriptHints.tsx

```diff
@@ -0,0 +1,37 @@
+export { HookTypeScriptHints }
+
+import React from 'react'
+import { assert, Link, Warning } from '@brillout/docpress'
+
+function HookTypeScriptHints({ hookTypeName }: { hookTypeName: `${string}Sync` | `${string}Async` }) {
+  let hookName: string
```

Comment:
So just to be clear, you'll have to lower-case the first letter of this

Reply:
Yea, I've seen. It's fixed (along with other issues 😅).

It works now and this time I actually tested it 🙂.